components/browser/session-storage/src/main/java/mozilla/components/browser/session/storage/BrowsingSession.kt

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.session.storage
+
+import mozilla.components.browser.state.state.recover.RecoverableTab
+
+/**
+ * A restored browsing session, read from disk.
+ */
+data class BrowsingSession(
```

Comment:
What do you think about renaming this to `RecoverableBrowserState`, similar to `RecoverableTab`? `BrowsingSession` keeps confusing me.

Reply:
Yeah, makes sense. For the other cases I wonder if we should slowly go away from "Session" and use "Tab".